### PR TITLE
Table 1

### PR DESCRIPTION
**Pull request by josephignace**
Format table 1 script for manuscripts (V1).

Note. The columns sizes need to be edited manually to fit the window.

### CODE REVIEW

File: analysis/figure/format_table1.R [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    orient = "landscape",
+    width = 8.3, height = 11.7
+  ),
+  type = "continuous",
+  page_margins = page_mar()
+)
+
+#Save table 1
+save_as_docx(table1_format, path = paste0(output_dir, "table1_neuro.docx"), pr_section = sect_properties)
+
```
Comment (marwaalarab) [inline analysis/figure/format_table1.R:79]: maybe name it table1_formatted.docx or another generic name that can be used in other repos.

File: analysis/figure/format_table1.R [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+#Notes 
+#While using word:
+#1. Change margins to narrow.
+#2. Merge vertically Characteristics and Sub characteristics with the upper cell, respectively.
+#3. Merge horizontally Pre-vaccination, vaccionation and unvaccinated with the right cell, respectivelly.
+#4. IF theme_vanilla is not used, you will need to add borders between each covariate, or remove unnecessary lines.
+#5. Change font, CVD uses "Calibri".
```
Comment (marwaalarab) [inline analysis/figure/format_table1.R:86]: refering to CVD paper the column name subcharacteristic should be removed.

**Review by marwaalarab — CHANGES_REQUESTED** [review]

File: analysis/figure/format_table1.R [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+library(data.table)
+library(tidyverse)
+library(flextable)
+library(officer)
+library(scales)
+
+#Directories
+results_dir <- "C:/Users/rs22981/Opensafely/Neuro/Release/20230408_tables/"
+output_dir <- "C:/Users/rs22981/Opensafely/Neuro/Release/20230408_tables/"
+
```
[the remark this answers is not in the document]
Comment (josephignace) [reply analysis/figure/format_table1.R:18]: Yes, it was just for testing. I will add a comment there, so everyone change it according to their repo.

File: analysis/figure/format_table1.R (hunk 1 above)
Comment (josephignace) [reply analysis/figure/format_table1.R:79]: I will use your suggestion, thanks.

File: analysis/figure/format_table1.R (hunk 2 above)
Comment (josephignace) [reply analysis/figure/format_table1.R:86]: I was not sure about it, but it makes sense. Thanks.

**Review by marwaalarab — APPROVED** [review]

**Pull request merged**